docs/documentation.md

```diff
@@ -46,46 +62,127 @@ All request should have in URL `stationId` param encoded with Base64.
+      "description": "This is the second task",
+      "priority": "MEDIUM",
+      "isDone": False,
+      "date": "2023-11-15",
+      "notificationTime": "10:00:00"
+    }
+    ``
+    
 - `PATCH /tasks/{taskId}` - Update a specific task.
     - Request
+
+    ```json
+    {
+      "title": "new task",
+      "description": "This is the new task",
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32" // optional
+    }
+    ```
+
+    - Response
     
     ```json
     {
-      "title": "Updated Task",
-      "description": "Updated details about the task",
-      "isDone": false,
-      "priority": "MEDIUM",
-      "date": "2023-11-15"
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "new task",
+      "description": "This is the new task", 
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32"
+    }
+    ```
+    
+- `PATCH /tasks/{taskId}` - Update a specific task.
+    - Request
+
+    ```json
+    {
+      "title": "Updated task2", // optional
+      "description": "This is the Updated task", // optional
+      "priority": "HIGH", // optional
+      "isDone": true, // optional
+      "date": "2022-12-23", // optional
+      "notificationTime": "12:04:32" // optional
+    }
+    ```
+
+    - Response
+    
+    ```json
+    {
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "Updated task2",
+      "description": "This is the Updated task",
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32"
     }
     ```
     
 - `DELETE /tasks/{taskId}` - Delete a specific task.
+    - Response:
+
+    ```json
+    {
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "Updated task2",
+      "description": "This is the Updated task",
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32"
+    }
+    ```
     
 **Statistics:**
 
-- `GET /statistics` - Get statistics for the authenticated user.
+- `GET /statistics` - Get statistics for the authenticated user for a specific date.
+    - Request
+
+    ```json
+    {
+      "Date": "2022-12-23"
+    }
+    ```
+
     - Response
     
     ```json
     {
-      "amountOfToDo": 15,
-      "amountOfDone": 15,
-      "amountOfAllTasks": 30,
-      "progress": 50 // amountOfDone / amountOfAllTasks
+      "Date": "2022-12-23",
+      "LifeBalanceValue": 100,
+      "Progress": null,
+      "FinishedTasks": 5,
+      "ToDo": 0,
+      "AllTasks": 5
     }
     ```
     
-- `GET /statistics/{date}` - Get statistics for the authenticated user for a specific date.
-
-```json
-{
-  "dateTime": "2023-11-21",
-  "amountOfToDo": 15,
-  "amountOfDone": 15,
-  "amountOfAllTasks": 30,
-  "lifeBalanceValue": 45 // weighted average of all tasks (%)
-}
-```
+- `GET /statistics` - Get statistics for the authenticated user.
+
+  ```json
+  {
+      "Date": null,
```

Comment:
I think `Date` is not needed here. Also if `LifeBalanceValue` is also not provided, it shouldn't appear here